docs: add changelog 4.22.0 (#36699)
ant-design/ant-design: CHANGELOG.en-US.md

---

## 4.22.0

`2022-07-26`

- Form
  - 🔥 Form support `Form.Item.useStatus` for custom components to get Form.Item validate status. [#36486](https://github.com/ant-design/ant-design/pull/36486)
  - 🆕 Form support `setFieldValue` to simplify config array index value. [#36058](https://github.com/ant-design/ant-design/pull/36058)
  - 🐞 Fix Form.Item shaking when trigger validate so fast. [#36575](https://github.com/ant-design/ant-design/pull/36575)

> - 🆕 Radio.Group support `ref.focus()` and `ref.blur()`. [#36041](https://github.com/ant-design/ant-design/pull/36041)

Reviewer: ```suggestion
- 🆕 Radio.Group support `onBlur` and `onFocus` props. [#36041](https://github.com/ant-design/ant-design/pull/36041)
```

- 🆕 Typography `ellipsis.tooltip` supports a object for Tooltip props. [#36099](https://github.com/ant-design/ant-design/pull/36099)
- 🛠 Refactor Drawer to remove directly style control which helps more React way. [#36672](https://github.com/ant-design/ant-design/pull/36672)
- 🛠 Refactor Sketelon.Button square shape style that its width is equal to height, and old become the default. [#36123](https://github.com/ant-design/ant-design/pull/36123) [@alanhaledc](https://github.com/alanhaledc)
- 🐞 Fix Modal.confirm `onCancel` argument close is not a function sometimes. [#36600](https://github.com/ant-design/ant-design/pull/36600) [@Wxh16144](https://github.com/Wxh16144)

> - 🐞 Fix Upload removing file status color is wrong. [#36706](https://github.com/ant-design/ant-design/pull/36706)

Author: ```suggestion
- 🐞 Revert [#36439](https://github.com/ant-design/ant-design/pull/36439) to fix the problem of incorrect status when uploading and deleting files, and fix the status color change when Upload removes files again change problem. [#36706](https://github.com/ant-design/ant-design/pull/36706)
```

> - 🐞 Fixed Upload removing file status color is wrong. [#36706](https://github.com/ant-design/ant-design/pull/36706)

Author: ```suggestion
- 🐞 Fix Upload removing file status color is wrong. [#36706](https://github.com/ant-design/ant-design/pull/36706)
```
